swap_meet/clothing.py

```diff
@@ -1,2 +1,19 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
```

Comment:
It's great that you made Clothing a child of the Item class! However, you're not fully taking advantage of that relationship. Item classes also hold on to category, condition, and age. Instead of setting self here, you can pass those values up to the Item class, which will do that for you. It lets you to get rid of those lines of code in all the child classes.